packages/electron-updater/src/providers/GitHubProvider.ts

```diff
@@ -77,17 +104,33 @@ export class GitHubProvider extends BaseGitHubProvider<GithubUpdateInfo> {
       throw newError(`No published versions on GitHub`, "ERR_UPDATER_NO_PUBLISHED_VERSIONS")
     }
 
-    const channelFile = getChannelFilename(this.getDefaultChannelName())
-    const channelFileUrl = newUrlFromBase(this.getBaseDownloadPath(tag, channelFile), this.baseUrl)
-    const requestOptions = this.createRequestOptions(channelFileUrl)
     let rawData: string
+    let channelFile: string = ""
```

Comment:
This is being flagged by eslint.
Please remove explicit type (since it can be assumed automatically)
`let channelFile = ""`

Reply:
In fact, is it possible to just move these to `let` vars into the `fetchData` function?
```
const fetchData = async (channelName: string) => {
      const channelFile = getChannelFilename(channelName)
      const channelFileUrl = newUrlFromBase(this.getBaseDownloadPath(String(tag), channelFile), this.baseUrl)
```